programmers-coding-test/wonji/greedy/큰수의법칙.js

```diff
@@ -0,0 +1,21 @@
+let N = prompt(`배열의 크기`);
+let M = prompt(`숫자가 더해지는 횟수 입력`);
+let K = prompt(`중복으로 숫자 더할 횟수 입력`);
+let data = prompt(`배열의 크기만큼 숫자들을 입력해주세요
```

Comment:
변하면 안되는 값들이므로 const로 선언하는 것이 좋겠다.

띄어쓰기 금지라는 조건을 준 이유는?
그렇게 되면 한 자릿수만 처리하게 된다.
애초에 문제를 풀 수 없는 것.
```suggestion
const N = prompt(`배열의 크기`);
const M = prompt(`숫자가 더해지는 횟수 입력`);
const K = prompt(`중복으로 숫자 더할 횟수 입력`);
const data = prompt(`배열의 크기만큼 숫자들을 입력해주세요
```